Client/Frontend/Sync/SyncPairCameraViewController.swift

```diff
@@ -181,11 +142,112 @@ class SyncPairCameraViewController: SyncViewController {
         }
     }
     
-    @objc func SEL_enterWords() {
+    @objc
+    private func onEnterWordsPressed() {
         let wordsVC = SyncPairWordsViewController(syncAPI: syncAPI)
-        wordsVC.syncHandler = self.syncHandler
+        wordsVC.delegate = delegate
         navigationController?.pushViewController(wordsVC, animated: true)
     }
+    
+    private func onQRCodeScanned(data: String) {
+        // Guard against multi-scanning
+        if cameraLocked { return }
+        cameraLocked = true
+        
+        // Pause scanning
+        cameraView.cameraOverlaySuccess()
+        cameraView.stopRunning()
+        
+        // Vibrate.
+        AudioServicesPlayAlertSound(SystemSoundID(kSystemSoundID_Vibrate))
```

Comment:
I believe this will play a sound if vibration is off `AudioServicesPlaySystemSound`. Just check if it is what we wanted.

Reply:
This is the original code that's always been there, I just moved it :)